src/midi/midi_fluidsynth.cpp

```diff
@@ -439,7 +487,7 @@ void MidiHandlerFluidsynth::Close()
 	synth.reset();
 	settings.reset();
 	last_played_frame = 0;
-	selected_font = "";
+	selected_font.clear();
```

Comment:
Nice, @johnnovak 
Thanks for taking the iron'ing press and cleaning up a bigger swath of PVS Studio issues in these files.

Reply:
Yeah, I'd like to see all of them gone sooner rather than later...